addons/mail/static/src/core/web/follower_list.xml

```diff
@@ -1,47 +1,38 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
 
-<t t-name="mail.FollowerList">
-    <t t-if="props.thread.hasWriteAccess">
-        <DropdownItem onSelected="() => this.onClickAddFollowers()" attrs="{ href: '#' }">
-            Add Followers
+    <t t-name="mail.FollowerList">
+
+        <DropdownItem class="'d-flex justify-content-between p-0'">
+            <t t-if="props.thread.selfFollower">
+                <a class="o-mail-unfollow-button d-flex align-items-center flex-grow-1 px-3 o-min-width-0 text-danger" href="#" role="menuitem" t-on-click="onClickUnfollow">
+                    Unfollow
+                </a>
+                <button class="o-mail-Follower-action btn btn-icon rounded-0" title="Edit subscription" aria-label="Edit subscription" t-on-click="(ev) => this.onClickEdit(ev)">
+                    <i class="fa fa-pencil"/>
+                </button>
+            </t>
+            <a t-else="" class="o-mail-follow-button d-flex align-items-center flex-grow-1 px-3 o-min-width-0 text-success" href="#" role="menuitem" t-on-click="onClickFollow">
+                Follow
+            </a>
```

Comment:
Please follow (no pun intended) BEM convention for naming custom classnames, like `o-mail-FollowerList-followBtn`, so that traceability + no conflicts.